shared/src/store/action-reducers/transfer.ts

```diff
@@ -176,6 +180,16 @@ export namespace TransferActionReducers {
                 isPaused: false,
             };
 
+            element.metaPosition = {
+                type: 'transfer',
+                transfer: {
+                    startPoint: cloneDeepMutable(startPoint),
+                    targetTransferPointId,
+                    endTimeStamp: draftState.currentTime + duration,
+                    isPaused: false,
+                },
+            };
+
```

Comment:
```suggestion
            element.metaPosition = {
                type: 'transfer',
                transfer: element.transfer
            };

```

or, if my recommendation from above is implemented:

```suggestion
            element.metaPosition = {
                type: 'transfer',
                ...element.transfer
            };

```

Reply:
I would defer this to #612